parsers/CO.py

```diff
@@ -133,41 +138,51 @@ def fetch_production(
 
     df_recursos = None
     df_generation = None
-    target_arrow_in_tz = arrow.now()
+    target_datetime_in_tz = datetime.now()
 
     if target_datetime is None:
-        target_arrow_in_tz = arrow.now().floor("day").to(TZ).shift(days=-XM_DELAY_MIN)
+        target_datetime_in_tz = datetime.now().replace(
+            hour=0, minute=0, second=0, microsecond=0
+        ).replace(tzinfo=TZ) + timedelta(days=-XM_DELAY_MIN)
```

Comment:
Same here, the second replace should probably be `.astimezone()`.

The timedelta can also be writtern as `- timedelta(days=XM_DELAY_MIN)` instead to improve readability.